framework/src/packets/ip/v6/srh.rs

```diff
@@ -384,9 +388,23 @@ impl<E: Ipv6Packet> IpPacket for SegmentRouting<E> {
         self.envelope_mut().set_dst(dst)
     }
 
+    // From https://tools.ietf.org/html/rfc8200:
```

Comment:
can you link to https://tools.ietf.org/html/rfc8200#section-8.1? makes it easier to find if someone decides to dig around later.

Reply:
ah, didn't know i get could get specific link anchors haha.